src/zenml/integrations/airflow/__init__.py

```diff
@@ -30,7 +30,7 @@ class AirflowIntegration(Integration):
     """Definition of Airflow Integration for ZenML."""
 
     NAME = AIRFLOW
-    REQUIREMENTS = ["apache-airflow~=2.4.0"]
+    REQUIREMENTS = ["apache-airflow~=2.5.0"]
```

Comment:
Why was this change necessary? This version is for example not supported on some managed Airflow platforms, not tested if that is going to be an issue though

Reply:
If that's an issue, I'll switch it back. I think I was trying to solve a conflict with other packages but forgot which one exactly (there were so many...)

Reply:
done